### ISSUE TITLE
No possibility to set a custom number of epochs

### ISSUE THREAD

**Issue by majcheradam** [ultralytics/hub#666]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and [discussions](https://github.com/ultralytics/hub/discussions) and found no similar questions.

### Question

<img width="489" alt="image" src="https://github.com/ultralytics/hub/assets/63547690/0dbe00e1-5a3e-432b-9aad-01cc936c4071"></br>
Hi, I have not found the possibility of setting a custom number of epochs, 
- will such a function be introduced? 
- Is it worth to train a model more than 100 epochs?

**Comment by pderrenger** [comment]
@majcheradam hello! 😄

Thanks for bringing up your question about setting a custom number of epochs. Customizing the number of epochs for training isn't directly available via the UI but can be adjusted through the configuration file before starting your training process. For more detailed steps, please refer to our documentation at https://docs.ultralytics.com/hub.

Regarding training a model for more than 100 epochs, it really depends on your dataset and the complexity of the model. In some cases, training for more epochs can improve model accuracy, especially with a large and varied dataset. However, beyond a certain point, you might encounter diminishing returns or even overfitting. Monitoring your model's performance on a validation set can help you determine the optimal number of epochs.

If you need any more guidance, feel free to ask! Our community is here to help.

**Comment by sergiuwaxmann** [comment]
@majcheradam Hello!
Yes, it is possible to customize the number of epochs by opening the advanced model configuration tile on the second step of the Train Model dialog.
